R/dcflights21.R

```diff
@@ -13,14 +13,48 @@
 #'   \item{origin, destination}{Origin and destination airports.}
 #'   \item{date}{Date of departure.} 
 #'   \item{hour}{Time of scheduled departure by hour}
-#'   \item{dep_delay}{Departure delay, in
-#'     minutes. Negative values represent early departures/arrivals.}
 #'   \item{plane}{Plane tail number, giving a unique ID for each plane.} 
-#'   \item{air_time}{Scheduled amount of time spent in the air, in minutes.} 
-#'   \item{duration}{Distance between airports, in miles.} 
+#'   \item{distance}{Distance between airports, in miles.} 
+#'   \item{duration}{Scheduled amount of time spent in the air, in minutes.} 
```

Comment:
@simonpcouch Notice that I changed this in the original dataset. Can you confirm this is right now?